webpack/components/extensions/HostDetails/DetailsTabCards/SystemPropertiesCardExtensions.js

```diff
@@ -35,6 +37,71 @@ SystemPropertiesCardSubscription.defaultProps = {
   hostDetails: {},
 };
 
+export const SystemPropertiesCardVirtualization = ({ hostDetails }) => {
+  if (!hostDetails?.subscription_facet_attributes) return null;
+
+  const {
+    virtualGuests,
+    hypervisor,
+    registeredThrough,
+    virtualHost,
+  } = propsToCamelCase(hostDetails.subscription_facet_attributes);
+  const virtualGuestIds = `name ^ (${virtualGuests.map(guest => guest.name).join(' and ')})`;
+
+  return (
+    <>
+      {hypervisor &&
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Virtual guests')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          <a href={`/hosts?search=${encodeURI(virtualGuestIds)}`}>
+            <Label color="blue" className="virtual-guests-label">
+              {sprintf(__('%s guests'), virtualGuests.length)}
+            </Label>
+          </a>
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+    }
+      {(!hypervisor || registeredThrough) &&
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Content source')}</DescriptionListTerm>
```

Comment:
Just realized that 'Registered through' / 'Content source' is already on the Registration card.

What do you think about eliminating it here, but rename the Registration card field to 'Content source'?

Reply:
Done.